tests/pytests/unit/modules/test_postgres.py

```diff
@@ -47,4 +67,68 @@ def idfn(val):
     ids=idfn,
 )
 def test_verify_password(role, password, verifier, method, result):
+    patcher = patch("salt.utils.path.which", MagicMock(return_value="/usr/bin/pgsql"))
```

Comment:
Since you didn't enter into a `with` statement, this patch is not really doing anything, right?
Or, if it is, it's not getting cleaned at the end, right?

Reply:
whoops... that was a rogue copy/paste...